storage/src/test/java/tech/pegasys/teku/storage/client/StorageBackedRecentChainDataTest.java

```diff
@@ -55,8 +57,10 @@
 
 public class StorageBackedRecentChainDataTest {
 
-  private static final BeaconState INITIAL_STATE =
-      new DataStructureUtil(3).randomBeaconState(UInt64.ZERO);
+  private static SpecProvider specProvider = StubSpecProvider.createMainnet();
+  private static final DataStructureUtil dataStructureUtil =
+      new DataStructureUtil(3, Optional.of(specProvider.getGenesisSpec()));
+  private static final BeaconState INITIAL_STATE = dataStructureUtil.randomBeaconState(UInt64.ZERO);
```

Comment:
I know it will be long, but can we inline the `dataStructureUtil` definition again please?  A `static DataStructureUtil` is a bad thing because then we lose the deterministic behaviour as it can wind up shared by multiple tests.

If it's really too long, might need to extract a private static method that creates the random beacon state.